lib/geolocation/simulator.js

```diff
@@ -18,16 +18,16 @@
 
 var console = require('console')
 
-var Simulator = function () {
+function Simulator() {
```

Comment:
I thought it was same.. maybe not if using prototypes ?

Reply:
It's almost the same, I think the difference mainly lies in hoisting, and in the function name being attached to the trace. I made the change because ESLint complained about it, though.

Reply:
eslint knows better than us ;)